x-pack/test/api_integration/apis/lens/existing_fields.ts

```diff
@@ -22,7 +22,6 @@ const fieldsWithData = [
   '@timestamp',
   '_id',
   '_index',
-  '_source',
```

Comment:
_source was removed from the API response, but this has no effect on Lens because it was hidden anyway.